apps/rahat-community/src/assets/email-templates/wallet-verification.hbs

```diff
@@ -1,21 +1,85 @@
-<html lang='en'>
+<!DOCTYPE html>
+<html lang="en">
 
-  <head>
-    <meta charset='UTF-8' />
-    <meta name='viewport' content='width=device-width, initial-scale=1.0' />
-    <title>Wallet Verification</title>
-  </head>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Wallet Verification</title>
+  <style>
+    body {
+      font-family: Arial, sans-serif;
+      background-color: #f5f5f5;
+      color: #333333;
+      margin: 0;
+      padding: 0;
+    }
+    .container {
+      max-width: 600px;
+      margin: 0 auto;
+      background-color: #ffffff;
+      padding: 20px;
+      border-radius: 5px;
+      box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+    }
+    .header {
+      text-align: center;
+      margin-bottom: 20px;
+    }
+    .header img {
+      max-width: 150px;
+    }
+    .content {
+      font-size: 16px;
+      line-height: 1.6;
+    }
+    .cta-button {
+      display: inline-block;
+      background-color: #4CAF50;
+      color: #ffffff;
+      padding: 12px 24px;
+      text-decoration: none;
+      border-radius: 4px;
+      font-weight: bold;
+      margin: 20px 0;
+    }
+    .cta-button:hover {
+      background-color: #45a049;
+    }
+    .footer {
+      font-size: 12px;
+      color: #777777;
+      text-align: center;
+      margin-top: 20px;
+      padding-top: 10px;
+      border-top: 1px solid #dddddd;
+    }
+    .footer a {
+      color: #555555;
+      text-decoration: none;
+    }
+  </style>
+</head>
 
-  <body>
-    <div class='container'>
+<body>
+  <div class="container">
+    <div class="header">
+      <img src="https://www.example.com/logo.png" alt="Company Logo">
+      <h2>Verify Your Wallet</h2>
+    </div>
+    
+    <div class="content">
       <p>Hi {{name}},</p>
-      <p>
-        Here is your Wallet Verification Link
-      </p>
+      <p>To complete the wallet registration process, please verify your wallet by clicking the button below:</p>
+      <a href="{{url}}" class="cta-button">Verify My Wallet</a>
```

Comment:
@dhirajchaurasiya10 , button background color should be rahat specific . Here is the color code : 
HEX#: #2B7EC1
RGB: 43, 126, 193
CMYK: 81, 45, 0, 0